.github/workflows/native-static-images.yml

```diff
@@ -0,0 +1,42 @@
+name: native-static-images
+on:
+  push:
+    paths:
+      - 'native-static-images/**'
+      - '.github/workflows/native-static-images.yml'
+  pull_request:
+    paths:
+      - 'native-shared-library/**'
+      - '.github/workflows/native-static-images.yml'
+  schedule:
+    - cron: "0 0 1 * *" # run every month
+  workflow_dispatch:
+permissions:
+  contents: read
+  pull-requests: write # for `native-image-pr-reports` option
+jobs:
```

Comment:
@LesiaChaban, you haven't setup the musl toolchain for the workflow. Compare with this workflow: https://github.com/graalvm/graalvm-demos/blob/master/.github/workflows/tiny-java-containers.yml

Reply:
I used  this command `native-image-musl: 'true'` for setup the musl toolchain.